libs/core/models/pagination.ts

```diff
@@ -0,0 +1,9 @@
+/** Pagination meta info. */
+export interface Pagination<T> {
+
+	/** Total count of items. */
+	readonly totalCount: number;
+
+	/** Array of items requested. */
+	readonly results: T[];
```

Comment:
Prefer to use `readonly` arrays where possible.
```suggestion
	readonly results: readonly T[];
```

Reply:
Why though?
According to our [wiki](https://wiki.saritasa.rocks/frontend/languages/ts-js/best-practices#promote-immutability) we should as much as possible to promote immutability.
So if `readonly` doesn't break things it's better to use it.
So I think previous comment was incorrect. It was a good idea to ask about clarification and motivation behind omitting `readonly`.